### ISSUE TITLE
Intensive testing the basic fusion software 

### ISSUE THREAD

**Issue by kyang2014** [TerraFusion/basicFusion#85]
We need at least to test 10 orbits , 30 orbits are desirable for the basic fusion software to if it can handle all the corner cases that we may possibly encounter. This is required before proceeding the final operation.

**Comment by LandonTClipp** [comment]
I am working on sorting out the current data on ROGER. Data is still very patchy/messy. Having trouble sorting the files in the proper order. I'm working with David on this.

**Comment by LandonTClipp** [comment]
I am making significant progress into getting the database query finished. I have found many missing files on ROGER, I have talked to Sean about that. I repurposed code from my old generateInput.sh script that orders the files returned by David's DB properly. Will still need to add the verification/error checking steps into this database query script I am developing (I can use old code for that too).
